client/layout/global-sidebar/style.scss

```
@@ -187,12 +187,16 @@ $brand-text: "SF Pro Text", $sans;
 					}
 				}
 
-				&.svg_all-sites,
-				&.sidebar_svg-reader {
+				&.svg_all-sites {
 					.is-global-sidebar-visible:not(.is-global-sidebar-collapsed) & {
 						margin-left: -2px;
 					}
 				}
+				&.sidebar_svg-reader {
+					.is-global-sidebar-visible:not(.is-global-sidebar-collapsed) & {
+						margin-left: 2px;
```

Could we try instead:
```suggestion
						margin-right: 10px;
```

Otherwise, the icon will also get pushed to the right a bit. Thanks!